src/roles.json

```diff
@@ -1533,18 +1533,18 @@
   },
   {
     "id": "lilmonsta",
-    "name": "Lil Monsta",
+    "name": "Lil' Monsta",
     "edition": "",
     "team": "demon",
-    "firstNight": 14,
+    "firstNight": 0,
     "firstNightReminder": "Wake all Minions together, allow them to vote by pointing at who they want to babysit Lil' Monsta.",
-    "otherNight": 33,
+    "otherNight": 0,
```

Comment:
This doesn't look right? Either it should be non-zero or there doesn't need to be a reminder text.